packages/@adobe/spectrum-css-temp/components/tags/index.css

```diff
@@ -17,7 +17,13 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Tags {
-  display: inline;
+  display: contents;
```

Comment:
Unfortunately I think this still breaks accessibility in Safari. https://bugs.webkit.org/show_bug.cgi?id=239479 When I tested it announces the table as having 0 columns and 0 rows and you cannot navigate to the cells with the voiceover cursor, whereas before it announced correctly.